.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,10 @@
+{
+    "configurations": [
+        {
+            "name": "Mac",
```

Comment:
could this be "macOS" instead?

Reply:
This is somehow used as an identifier by vscode, so i would stick to it, see

<img width="1127" alt="image" src="https://github.com/user-attachments/assets/ee71af39-89a5-4d70-9fbe-554695cfecf4" />